### PR TITLE
fix: remove version lock of @faker-js/faker [PLAT-2487]

### PR DESCRIPTION
**Pull request by prescottprue**
:tickets: TICKET-000 

## Changes
Admin-ui is on v8 with no issues - this skip doesn't seem relevant anymroe

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -93,11 +93,6 @@
       "matchPackageNames": ["faker", "@types/faker"],
       "enabled": false
     },
-    {
-      "description": "Skip @faker-js/faker update to v7 (since it is ESM) [PLAT-2487]",
-      "matchPackageNames": ["@faker-js/faker"],
-      "allowedVersions": "<=6.3.1"
```
Comment (hershmire) [inline default.json:99]: It might be fine for our UIs but won't this be problematic for services that are not ESM?

Comment (prescottprue) [reply default.json:99]: Identity-service is already on v7 and there doesn't seem to be an issue there: https://github.com/reside-eng/identity-service/blob/main/package.json#L71

**Review by hershmire — APPROVED** [review]

File: default.json (hunk 1 above)
Comment (hershmire) [reply default.json:99]: Okay, let's drop this then.

**Comment by prescottprue** [comment]
I was also able to update core-functions to v9 of faker without issue

**Pull request merged**